File: 프로그래머스/unrated/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.java
class Solution {
    public String solution(String my_string, String alp) {
        String answer = "";
        for (int i = 0; i < my_string.length(); i++) {
            char currentChar = my_string.charAt(i);
            if (String.valueOf(currentChar).equals(alp)) {
                answer += String.valueOf(currentChar).toUpperCase();
            } else {
                answer += currentChar;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.js
function solution(my_string, alp) {
    let answer = "";
    for (let i = 0; i < my_string.length; i++) {
        let currentChar = my_string.charAt(i);
        if (currentChar === alp) {
            answer += currentChar.toUpperCase();
        } else {
            answer += currentChar;
        }
    }
    return answer;
}